pkg/actions/nodegroup/get.go

```diff
@@ -71,7 +71,7 @@ func (m *Manager) GetAll() ([]*manager.NodeGroupSummary, error) {
 			CreationTime:         describeOutput.Nodegroup.CreatedAt,
 			NodeInstanceRoleARN:  *describeOutput.Nodegroup.NodeRole,
 			AutoScalingGroupName: strings.Join(asgs, ","),
-			Version:              *describeOutput.Nodegroup.Version,
+			Version:              getOptionalValue(describeOutput.Nodegroup.Version),
```

Comment:
can we add a test for this if possible? 💡

Reply:
We need to get this PR out today so I'll merge it now and look into adding tests later.